### PR TITLE
Added Build Systems Section

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@patriksvensson @devlead looks like the Build System section was already there.  Just needed to be exposed :smile:

File: src/Cake.Web/App_Data/docs.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <item title="Visual Studio Code" file="editors/vscode.md" />
   </section>
   <section id="build-systems" title="Build Systems">
-    <item title="Visual Studio Online" file="build-systems/vso.md" hidden="true"/>
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs.xml]: Not sure we can change the URI of this one since we link to it from the marketplace.

Comment (gep13) [inline src/Cake.Web/App_Data/docs.xml]: @patriksvensson ah, good catch! I have changed this back.

Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs.xml]: There is support in the site for legacy redirects. Maybe you can add it there and just redirect it to the new URI?

Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs.xml]: Have a look here: https://github.com/cake-build/website/blob/develop/src/Cake.Web/App_Start/LegacyRouteConfig.cs

Comment (gep13) [inline src/Cake.Web/App_Data/docs.xml]: I don't think it is a big deal.  I was just being pedantic, and thought I would change it while I was there.  I am happy to leave it as is.

Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs.xml]: @gep13 Absolutely!

**Comment by patriksvensson** [comment]
Look good to me. Going to merge this then! :+1:

**Comment by patriksvensson** [comment]
Related to #82.

**Pull request merged**